test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -1383,6 +1395,29 @@ TEST_F(SingleBalancerTest, BackendsRestart) {
   EXPECT_EQ(1U, balancers_[0]->service_.response_count());
 }
 
+TEST_F(SingleBalancerTest, TargetFromLbPolicyConfig) {
+  constexpr char kServiceConfigWithTarget[] =
+      "{\n"
+      "  \"loadBalancingConfig\":[\n"
+      "    { \"grpclb\":{\n"
+      "      \"serviceName\":\"test_target\"\n"
+      "    }}\n"
+      "  ]\n"
+      "}";
+
+  SetNextResolutionAllBalancers(kServiceConfigWithTarget);
+  const size_t kNumRpcsPerAddress = 1;
+  ScheduleResponseForBalancer(
+      0, BalancerServiceImpl::BuildResponseForBackends(GetBackendPorts(), {}),
+      0, "test_target");
+  // Make sure that trying to connect works without a call.
+  channel_->GetState(true /* try_to_connect */);
+  // We need to wait for all backends to come online.
+  WaitForAllBackends();
+  // Send kNumRpcsPerAddress RPCs per server.
+  CheckRpcSendOk(kNumRpcsPerAddress * num_backends_);
```

Comment:
This can probably be removed; it doesn't really add anything for this particular test.

Reply:
If we don't send RPCs, the channel is not connected and no load balancing request is ever made, right?

I changed it to send 1 RPC instead of mulitple, since that should be enough.

Reply:
The channel will connect before we even send a single RPC, because of the call to `GetState(true)` above.  Also, the call to `WaitForAllBackends()` will send RPCs as well.  It is not necessary to send any more RPCs.